lvmd/command/lvm.go

```diff
@@ -44,16 +44,20 @@ func callLVM(ctx context.Context, cmd string, args ...string) error {
 // cmd is a name of sub-command.
 func callLVMWithStdout(ctx context.Context, cmd string, args ...string) ([]byte, error) {
 	var stdout bytes.Buffer
+	var stderr bytes.Buffer
 	args = append([]string{cmd}, args...)
 
 	c := wrapExecCommand(lvm, args...)
 	c.Env = os.Environ()
 	c.Env = append(c.Env, "LC_ALL=C")
 	c.Stdout = &stdout
-	c.Stderr = os.Stderr
+	c.Stderr = &stderr
 
 	log.FromContext(ctx).Info("invoking LVM command", "args", args)
 	err := c.Run()
+
+	log.FromContext(ctx).Info("result of LVM command", "stdout", stdout.Bytes(), "stderr", stderr.Bytes())
```

Comment:
Can we introduce this at a lower log level? I do not think this is necessary for common debugging

Reply:
I would argument for V(2) at the very least for both commands then but maybe we put that into a separate issue as it affects pretty much all commands? WDYT @pluser @toshipp ?

Reply:
Overall what is logged and when could use some improvement its not "bad" but its not very intentional, for example at the info level I would like to know what vgs were found on startup so in the list function we should have a dump of the VGs, as LVs actions we should see that at info level. this really should be debug but as a non go programmer I wasn't really wanting to do more than I could safely copy and paste.

Reply:
I agree with @jakobmoellerdev . Please separate it.